ui/README.md

```diff
@@ -1,22 +1,41 @@
 # Setting up MAAS UI
 
-**Note**: You will need an instance of MAAS running in the same container in order to run MAAS UI.
+**Note: You will need an instance of MAAS running in order to run MAAS UI.**
 
-**Note**: You will need your public and private SSH keys inside your container to be able to clone the repo. For help with this see [ssh-copy-id](https://www.ssh.com/ssh/copy-id).
+## Generate an SSH key in your container (LXD and multipass)
 
-Inside your MAAS container (either LXD or multipass) go to the parent directory of your MAAS installation, clone the MAAS UI repo and set it's upstream and origin:
+**Note: The following instructions should be run in the same container that also contains your local MAAS.**
+
+Inside your container (LXD or multipass) [generate a new SSH key](https://help.github.com/en/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) and [add it to your Github account](https://help.github.com/en/articles/adding-a-new-ssh-key-to-your-github-account).
```

Comment:
@steverydz can you explain why we're recommending this? If someone already has a key, they should copy it to the container.

Reply:
Ultimately it's because it's more straightforward, especially for those who aren't developers because they either have to copy and paste then figure out how to set the right permissions or try to use `ssh-copy-id` which always fails because there is no public key.